### ISSUE TITLE
CloudyGrid on the BC03 bare-stellar SSP returns an all-NaN sed_nebular at every grid node (PRSC grid pairs only with the PRSC FSPS SSP)

### ISSUE THREAD

**Issue by suchethac** [suchethac/tengri#2418]
**Measured on main (5a4eec87f) and on the #2346 branch alike:** a `CloudyGrid` model built on the bare-stellar BC03 SSP (`bc03_pdva_stelib_chabrier.h5`) with the shipped `cloudy_grid_prsc.h5` returns `sed_nebular` = NaN at every one of the 7955 master-grid nodes, not only past the grid edge. The same build on the PRSC-matched FSPS SSP (`fsps_prsc_miles_chabrier.h5`) is finite everywhere.

Repro (worktree or main checkout, `PYTHONPATH=<checkout>/src`):

```python
import jax, numpy as np, tengri
ssp = tengri.load_ssp_data("data/bc03_pdva_stelib_chabrier.h5")
m = tengri.SEDModel.build(
    ssp_data=ssp,
    sfh={"type": "const", "all_params": tengri.Fixed(tengri.DEFAULT)},
    dust_attenuation={"type": "two_component", "law": "calzetti", "all_params": tengri.Fixed(tengri.DEFAULT)},
    dust_emission={"type": "draine_li2014", "all_params": tengri.Fixed(tengri.DEFAULT)},
    neb={"type": "cloudy", "all_params": tengri.Fixed(tengri.DEFAULT)},
    radio={"sf": {"type": "bell2003"}, "agn": {"type": "powerlaw"}, "all_params": tengri.Fixed(tengri.DEFAULT)},
    redshift=tengri.Fixed(0.0),
)
st = m.predict_state(dict(m.spec.sample(jax.random.PRNGKey(0))))
print(np.isnan(np.asarray(st.derived["sed_nebular"])).sum(), "of", st.wave.size)   # 7955 of 7955
```

The grid file itself is clean: `continuum/luminosity` has shape (11, 10, 7, 1963), 0 NaN, 0 inf, min 0, max 7.0e-60; `continuum/wavelength` runs 91 Å to 1e8 Å. So the NaN is produced in the backend's (met, age, logU) lookup or the Q_H scaling for this SSP, not read from disk. Candidate: BC03's `ssp_lgmet` / `ssp_lg_age_gyr` axes fall outside the PRSC grid's `log_met` / `log_age_yr` axes and the bilinear index/weight helper returns NaN instead of clamping or raising; a silent all-NaN SED is the #302/#1099 silent-failure class.

Surfaced while writing `tests/regression/bug/test_bug_2346_nebular_freefree_one_thermal_term.py::test_cloudy_grid_continuum_carries_the_same_tail`, which therefore pairs the PRSC grid with the PRSC FSPS SSP. Expected fix shape: either clamp/extrapolate the SSP axes onto the grid with a warning, or refuse the mismatched (SSP, grid) pair at build time with a message naming both axes; plus a regression test that a CloudyGrid model on every shipped bare-stellar SSP returns a finite `sed_nebular`.

**Comment by suchethac** [comment]
Claiming this in the issues-review wave (orchestrator session "review tengri issues"): measure-first lane to localize the first NaN in the CloudyGrid path on the BC03 SSP, then a RED-first fix on a synthetic reproduction of the same condition.

**Comment by suchethac** [comment]
Diagnosed and fixed in #2459. The hypothesis in the issue text (a PRSC grid paired with a Padova-isochrone SSP) is not the cause: the eight shipped `cloudy_grid_*.h5` files share one axis set (log U −4..−1, log age/yr 5.70..7.30, log Z −1.98..0.2), and the PRSC grid + FSPS PRSC SSP pairing is byte-identical before and after the fix.

The cause is the SSP. `bc03_pdva_stelib_chabrier.h5` is the only shipped SSP with an age-0 anchor template, `ssp_lg_age_gyr[0] = -inf`. The three photoionized backends built their Q_H table's age axis as `ssp_lg_age_gyr + 9.0`, so the axis started at `-inf`; `_interp_index_weight` then saw an infinitely wide first interval and returned NaN weights for the anchor and for every query inside that interval (`_get_qh_at(-1.85, -inf)` → NaN, `(-1.85, 5.0)` → NaN, `(-1.85, 6.0)` → 0.639), and the per-age sum spread the NaN to all 7955 nodes and every line. The stellar path floors this anchor at 0.1 Myr (#1016); the nebular axes now use the same floor (`ssp_log_age_yr_axis`, `LOG_AGE_YR_FLOOR = 5.0`), and the shared interpolator treats a non-finite node as a zero-width interval. CB19 and MAPPINGS had the same defect on BC03, untested until now. Cue was immune only because it masks non-finite `log Q_H` per template.

After the fix the issue's exact build gives 0 NaN of 7955 nodes, finite max 1.64e30 erg/s/Hz, all 68 nodes above 1e8 Å positive.
